terraform/cloud-platform-components/README.md

```diff
@@ -82,6 +83,114 @@ Metrics-server allows us to perform resource queries against the cluster. Comman
 ## Nginx-ingress
 A vital component in the cluster. The Nginx-ingress controller is a daemon, deployed as a Kubernetes Pod, that watches the apiserver's /ingresses endpoint for updates to the Ingress resource. Its job is to satisfy requests for Ingresses.
 
+## Open-Policy-Agent
+
+OPA is a lightweight general-purpose policy engine that can be co-located with our service. Services offload policy decisions to OPA by executing queries. OPA evaluates policies and data to produce query results (which are sent back to the client).  
```

Comment:
I would suggest tweaking this a little bit:

> The [Open Policy Agent](https://www.openpolicyagent.org) (OPA) is a lightweight...

Reply:
Agree, updated accordingly.